src/app/docs/definition-list/definition-list-docs.component.html

```diff
@@ -5,6 +5,9 @@
 >
   <sky-docs-demo-page-summary>
     Definition lists display label-value pairs.
```

Comment:
```suggestion
    <p>
      Definition lists display label-value pairs.
    </p>
```
We need the `p` tag around the opening paragraph now that we've added the alert to keep the paragraph and alert from being stacked too tightly.